src/schema/me/__tests__/recently_viewed_artworks.test.ts

```diff
@@ -105,7 +105,7 @@ describe("RecentlyViewedArtworks", () => {
       data: { recordArtworkView: { artwork_id: "percy" } },
     }
 
-    const mockFetch = fetch as jest.Mock<any>
+    const mockFetch = (fetch as unknown) as jest.Mock<any>
```

Comment:
typescript started complaining about this cast because I added the types for `node-fetch`